www-servers/caddy/caddy-2.7.5.ebuild

```diff
@@ -0,0 +1,98 @@
+# Copyright 1999-2023 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit fcaps go-module systemd shell-completion
+
+DESCRIPTION="Fast and extensible multi-platform HTTP/1-2-3 web server with automatic HTTPS"
+HOMEPAGE="https://caddyserver.com"
+
+if [[ "${PV}" == *9999* ]]; then
+	inherit git-r3
+	EGIT_REPO_URI="https://github.com/caddyserver/caddy.git"
+else
+	SRC_URI="https://github.com/caddyserver/caddy/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+	SRC_URI+=" https://github.com/rahilarious/gentoo-distfiles/releases/download/${P}/deps.tar.xz -> ${P}-deps.tar.xz"
+	SRC_URI+=" https://github.com/caddyserver/dist/archive/refs/tags/v${PV}.tar.gz -> ${P}-docs.tar.gz"
+	KEYWORDS="~amd64 ~arm64 ~loong ~riscv"
+fi
+
+LICENSE="Apache-2.0"
+LICENSE+=" BSD ECL-2.0 MIT CC0-1.0"
+SLOT="0"
+RESTRICT="test"
+RDEPEND="
+	acct-user/http
+	acct-group/http"
+DEPEND="${RDEPEND}"
+
+FILECAPS=(
+	-m 755 'cap_net_bind_service=+ep' usr/bin/"${PN}"
+)
+
+PATCHES=(
+	"${FILESDIR}"/remove-binary-altering-commands-2.7.5.patch
+)
+
+src_unpack() {
+	if [[ "${PV}" == *9999* ]]; then
+		# unpack code
+		git-r3_src_unpack
+
+		# unpack docs and misc
+		EGIT_REPO_URI="https://github.com/caddyserver/dist.git"
+		EGIT_CHECKOUT_DIR="${WORKDIR}/dist-${PV}"
+		git-r3_src_unpack
+
+		go-module_live_vendor
+	else
+		go-module_src_unpack
+	fi
+}
+
+src_prepare(){
+	default
+	sed -i -e "s|User=caddy|User=http|g;" ../dist-"${PV}"/init/*service || die
```

Comment:
You could combine this into one sed by adding two `-e`s

Reply:
done